scripts/dev-openbao-init.sh

```diff
@@ -0,0 +1,11 @@
+#!/bin/sh
+
+export TOKEN=$1
+echo "TOKEN: $TOKEN"
+
+bao login -method=token token=$TOKEN
+
+bao secrets enable transit
+bao write transit/keys/terraform-backend exportable=true
+
+sleep infinity
```

Comment:
This slows down stopping the container. Maybe we can remove this line, and add `restart: on-failure` the the docker-compose service.